app/invocation/invocation_cache_card.tsx

```diff
@@ -7,6 +7,8 @@ interface Props {
   model: InvocationModel;
 }
 
+const bitsPerByte = 8;
```

Comment:
nit: Do we want to use CAPITALS_WITH_UNDERSCORES for constants?

Reply:
Done